File: assets/code-variants/ch13/26.java
@Test
public void sortNumbers() {
    NumberSorter numberSorter = new NumberSorter(quicksort, bubbleSort);
    // Call the system under test.
    List sortedList = numberSorter.sortNumbers(newList(3, 1, 2));
    // Validate that the returned list is sorted. It doesn’t matter which
    // sorting algorithm is used, as long as the right result was returned.
    assertThat(sortedList).isEqualTo(newList(1, 2, 3));
}

File: assets/code-variants/ch13/26.py
def test_sort_numbers():
    number_sorter = NumberSorter(quicksort, bubble_sort)
    # Call the system under test.
    sorted_list = number_sorter.sort_numbers([3, 1, 2])
    # Validate that the returned list is sorted. It doesn't matter which
    # sorting algorithm is used, as long as the right result was returned.
    assert sorted_list == [1, 2, 3]
